src/entities/configuration/configuration.ts

```diff
@@ -15,12 +15,21 @@ export interface ConfigurationOOODay {
   }>
 }
 
+export interface AttributionModels {
+  last_touch: boolean
+  coupon_code_unstitched: boolean
+  coupon_code_all: boolean
+}
```

Comment:
Idea: If we have the AttributionModels as an enum somewhere here, then we can use that to generate this type:
```suggestion
export type AttributionModels = Record<AttributionModel, boolean>
```